### PR TITLE
Feat/126-update-file-metadata

### PR DESCRIPTION
**Pull request by saneDG**
🔔 This PR is pointing to #163 !

- Add update-metadata endpoint and lambda.

- Edit update-file endpoint -> use `../{nodeId}/content` path parameter for content update and `../{nodeId}/` for updating metadata.

### CODE REVIEW

File: packages/server/lambdas/alfresco/update-file-metadata.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    const res = await fetch(url, options);
+    const result = (await res.json()) as AlfrescoResponse;
+    return result;
+  } catch (err) {
+    console.error('error:' + err);
+  }
+};
+
+/**
+ * Upload custom content for page. Example request: /api/alfresco/file/linjakaaviot
```
Comment (NinaDang97) [inline packages/server/lambdas/alfresco/update-file-metadata.ts:32]: Same here "Update metadata for page"

**Review by NinaDang97 — APPROVED** [review]

File: packages/server/lambdas/alfresco/fileRequestBuilder/index.ts [hunk 2]
```diff
@@ -18,6 +18,14 @@ export const updateFileRequestBuilder = (event: ALBEvent, headers: HeadersInit,
       return alfrescoRequestBuilder.updateRequestBuilder(event, headers);
   }
 };
+export const updateFileMetadataRequestBuilder = (event: ALBEvent, headers: HeadersInit, store?: FileStore) => {
+  switch (store) {
+    case FileStore.ALFRESCO:
+    default:
```
Comment (NinaDang97) [inline packages/server/lambdas/alfresco/fileRequestBuilder/index.ts:24]: Wonder why these lines are not listed under `case FileStore.ALFRESCO:` but `default:`?

Comment (mehtis) [reply packages/server/lambdas/alfresco/fileRequestBuilder/index.ts:24]: Actually, it's both. With no break or return in a case, the switch won't stop going over the list. So this is kinda like `if (case === FileStore.ALFRESCO || case === default) { /* DO STUFF */}`

**Review by mehtis — APPROVED** [review]

File: packages/server/lambdas/alfresco/update-file-metadata.ts (hunk 1 above)
Comment (mehtis) [reply packages/server/lambdas/alfresco/update-file-metadata.ts:32]: ☝️

**Pull request merged**